lib/Data/Validate/Sanctions/Fetcher.pm

```diff
@@ -425,6 +431,95 @@ sub _eu_xml {
     };
 }
 
+sub _unsc_xml {
+    my ($xml_content) = @_;
+
+    # Preprocess the XML content to escape unescaped ampersands
+    $xml_content =~ s/&(?!(?:amp|lt|gt|quot|apos);)/&amp;/g;
+    my $data = XML::Simple::XMLin(
```

Comment:
why are we using a parcer here ? 
Can't we use `xml2hash` and reuse the functionalities we had before ? Check the _ofac_xml sub for reference